care/facility/models/patient.py

```diff
@@ -431,9 +431,6 @@ def __str__(self):
         return f"{self.name} - {self.year_of_birth} - {self.get_gender_display()}"
 
     @property
```

Comment:
@jasonmokk  you should probably remove `@property` decorator as well

Reply:
Got it, thanks for the review. PR has been updated.